info-compiler/src/test/java/com/google/sps/InfoCompilerTest.java

```diff
@@ -125,6 +128,15 @@ public void resetDatastore() throws IOException {
     infoCompiler = new InfoCompiler(datastore);
   }
 
+  @Test
```

Comment:
Can you add tests for how your code handles invalid data?

Reply:
Will do!

Reply:
Could you clarify what you mean by invalid data? Because the election and contest information comes from the Civic Information API, it would have a fixed structure. The data from the National Address Database also have a fixed structure.

In this commit: https://github.com/googleinterns/voter-central/pull/26/commits/5543ba9280f685d8a33cf87637ba0fc1987e7844, I added a test for the special Voter Info Project test election, which would be screened out.

Reply:
just wanted a test for something that should be screened out. thanks!